src/plugins/gui/providers/bityProvider.ts

```diff
@@ -191,7 +193,10 @@ const fetchBityQuote = async (bodyData: BityQuoteRequest) => {
     const newData = await result.json()
     return newData
   } else {
-    throw new Error('Bity: Unable to fetch quote: ' + (await result.text()))
+    const bityErrorRes = asBityErrorResponse(await result.json())
```

Comment:
The `result.json()` might throw if the result is not json. Wrap this in try/catch and add back the 

`throw new Error('Bity: Unable to fetch quote: ' + (await result.text()))`

Reply:
Are we guaranteed result.json() if result.ok? I was relying on
[fuzzyTimeout](https://github.com/EdgeApp/edge-react-gui/blob/a73f679ba2ba7603396641edc1a9052f92cf736b/src/plugins/gui/amountQuotePlugin.ts#L201)
to deal with such errors.

Reply:
Are we guaranteed `result.json()` if `result.ok`? As a fallback, I was relying on [`fuzzyTimeout`](https://github.com/EdgeApp/edge-react-gui/blob/a73f679ba2ba7603396641edc1a9052f92cf736b/src/plugins/gui/amountQuotePlugin.ts#L201), which would catch any of the throws from here.

Reply:
Oh nevermind, I see I removed that bity-worded error message

Reply:
Also added this as a note to handle better in a future task